examples/configurable/configurable.out

```diff
@@ -32,3 +31,15 @@ maximum payload (in MB): 0.321
 remote enabled: false
 admin details: {"username":"admin","password":"password"}
 users: [{"username":"John","password":"abc123"},{"username":"Bob","password":"cde456"}]
+
+# A similar configuration can be given via CLI parameters in the
+# following format.
+
+bal run configurable.bal -- -Chost=192\.168\.1\.1 -Cport=9091 -CenableRemote=false -CmaxPayload=0.456
```

Comment:
@warunalakshitha do we need to escape the dots?

Reply:
It seems we don't need to escape `.` character in CLI. Removed it in latest commit.